client/src/components/PickDate/PickDate.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { DateInput } from '@blueprintjs/datetime';
+import './PickDate.scss';
+
+class PickDate extends React.Component {
+  state = {
+    closeOnSelection: true,
+    date: null,
+    disabled: false
+  };
+
+  handleDateChange = date => {
+    this.setState({ date: new Date(date) });
+  };
+  render() {
+    return (
+      <div>
+        <div className="content">
+          <span> DueDate: </span>
+        </div>
+        <DateInput
+          formatDate={date => date.toLocaleString()}
```

Comment:
There should be some function that can be appended here to print only the date (without the 00:00 time)

Maybe append `substring(0,10)`?

Reply:
Check output of `new Date().toISOString().substring(0, 10)` in browser console.